Assets/Scripts/Duncan.cs

```diff
@@ -5,11 +5,15 @@
 public class Duncan : MonoBehaviour
 {
     public float speed = 4;
+
+    public void Init(Vector2 displacement){
+        GetComponent<Rigidbody2D>().velocity = displacement * (this.speed / displacement.magnitude);
```

Comment:
Ooh, okay, that's a tough one. GetComponent's a method inherited from MonoBehavior and it's not defined anywhere in Duncan.cs, so there's absolutely some small amount of utility to gain from putting "this." in front of it for clarity, but I've avoided putting "this." on method calls so far because I'm unsure where to draw the line.

Do we put "this." on every non-static method inherited by a class? What about the methods defined in that class that are called, should we be calling CalcDelayVector in Robo.cs with "this." prepended since it's a method tied to a single Robo instance, and we always define where the method comes from with "this."? If not, what about methods defined or declared in a superclass but overridden or implemented in the subclass? If we added a static method MethodName to Duncan.cs, do we call it as "Duncan.MethodName" within Duncan.cs to ensure we always refer to where the method is defined?

I'm asking seriously, if the answer to these questions is a resounding "yes, that will improve readability" to you, then I'm 100% on-board with always explicitly defining from what source a method is derived for clarity. No complaints from me. I just need to know where the line is.

My personal stance on the matter is that we should either do all of the above or none of the above, no in-between, for maximal consistency.

Reply:
Let's go explicit for all of them, then! I think the readability gained outweighs the cognitive load -- in fact, since we're all new, it may even save us some down the road :)

Reply:
Heck yeah. I'm not sure if committing a suggestion resolves the conversation automatically, and I don't want this resolved until I push a change later today that makes all the function calls more explicit, so I'll commit this suggestion myself as part of a broader commit that spans the various scripts covered in this branch.

Reply:
Pushed changes that included this suggestion right before the meeting today. And just added the change discussed to the MUGA, so this is now resolved.